librz/include/rz_il/rz_il_opbuilder_begin.h

```diff
@@ -99,12 +103,12 @@
 #define AND(x, y) rz_il_op_new_bool_and(x, y)
 #define OR(x, y)  rz_il_op_new_bool_or(x, y)
 
-#define FNEQ(flx, fly) rz_il_op_new_fneq(flx, fly)
-#define FEQ(flx, fly)  rz_il_op_new_feq(flx, fly)
-#define FLT(flx, fly)  rz_il_op_new_flt(flx, fly)
-#define FLE(flx, fly)  rz_il_op_new_fle(flx, fly)
-#define FGT(flx, fly)  rz_il_op_new_fgt(flx, fly)
-#define FGE(flx, fly)  rz_il_op_new_fge(flx, fly)
+#define FNE(flx, fly) rz_il_op_new_fneq(flx, fly)
```

Comment:
Do you think renaming `FNEQ()` helper macro to `FNE()` really necessary? @thestr4ng3r @DMaroo @Rot127 @wargio what do you think?

Reply:
ok with that but these changes also needs to be done in `rz_il_opbuilder_end.h`

Reply:
I really don't have a strong preference. But `FNE` does fit in nicer with all the other 3 letter floating-point comparison macros.